src/shortcuts.rs

```diff
@@ -243,17 +243,19 @@ macro_rules! id {
      };
 }
 
+// reference for workaround used by style! macro
+// (https://github.com/dtolnay/case-studies/blob/master/autoref-specialization/README.md)
 /// Provide a shortcut for creating styles.
 #[macro_export]
 macro_rules! style {
```

Comment:
Could you write tests for it?
e.g. `div![ style!{  St::Foo => bar  } ]` is equal to `<div style=".."></div>` where `bar` is `String`, `Some(T)`, `None`, `&str`, etc.

Reply:
where should I write these tests ?

Reply:
1. option - Add block with tests into `shortcuts.rs`  (you can get inspired by `css_units.rs`) and test only `style!`.
2. option - Add tests similar to https://github.com/David-OConnor/seed/blob/efd6b0b4ebcfcd860bb49fed702c54a103f302ad/src/dom_types.rs#L1387 into `dom_types.rs`.

I don't have strong preference, choose the most suitable option (or another better one).

Reply:
Done, Added to `dom_types.rs`